tests/testerrors.nim

```diff
@@ -1,80 +0,0 @@
-#[
-  This module uses debug versions of the rpc components that
```

Comment:
How come we don't consider the removal of these tests a regression?

Reply:
they were never run (not part of `tests/all.nim`) - they also didn't match the behavior of the existing code - neither before nor after these fixes - in particular, they described that the `error` field in `Response` was used, when in fact an exception was being raised.